core/src/test/scala/unit/kafka/server/ClientQuotasRequestTest.scala

```diff
@@ -212,7 +214,10 @@ class ClientQuotasRequestTest extends BaseRequestTest {
           InetAddress.getByName(unknownHost)
         else
           InetAddress.getByName(entityName)
-        assertEquals(expectedMatches(entity), servers.head.socketServer.connectionQuotas.connectionRateForIp(entityIp), 0.01)
+        TestUtils.retry(10000L) {
```

Comment:
It seems to me ```waitUntilTrue``` is more suitable since the ```exception``` is not necessary in this test case.

Reply:
Thank you